### PR TITLE
Make the title readable in generated schemas

### PR DESCRIPTION
**Pull request by Revolyssup**
Signed-off-by: ashish <ashishjaitiwari15112000@gmail.com>

**Description**

This PR fixes #

**Notes for Reviewers**

**[Signed commits](../CONTRIBUTING.md#signing-off-on-commits-developer-certificate-of-origin)**
- [x] Yes, I signed my commits.

### CODE REVIEW

File: utils/manifests/utils.go [hunk 1]
```diff
@@ -280,3 +280,38 @@ func deleteFile(path string) error {
 	}
 	return nil
 }
+
+//While going from Capital letter to small, insert a whitespace before the capital letter.
+//While going from small letter to capital, insert a whitespace after the small letter
```
Comment (leecalcote) [inline utils/manifests/utils.go:285]: Excellent.

Comment (leecalcote) [reply utils/manifests/utils.go:285]: //While going from Capital to Capital, do nothing.

Comment (Revolyssup) [reply utils/manifests/utils.go:285]: Yupss, that is inbuilt logic

**Review by leecalcote — APPROVED** [review]

File: utils/manifests/utils.go (hunk 1 above)
Comment (Revolyssup) [reply utils/manifests/utils.go:285]: Do nothing also, when small to small

**Comment by leecalcote** [comment]
Merge (and release, if needed) at-will.

**Comment by Revolyssup** [comment]
> Merge (and release, if needed) at-will.

I do not have read/write access

**Pull request merged**